targets/targets.json

```diff
@@ -495,6 +495,10 @@
         "release_versions": ["2", "5"],
```

Comment:
I think we typically only place the `release_versions` field in the "leaf" target, eg. the target that is the furthest down the inheritance chain. So in this case I would move this to the `K22F` target definition.